controller/src/main/java/org/jboss/as/controller/operations/global/ReadFeatureDescriptionHandler.java

```diff
@@ -122,16 +122,16 @@
 
     private final ImmutableCapabilityRegistry capabilityRegistry;
 
-    private static final String $PROFILE_PREFIX = "$profile.";
-    private static final String ADDRESS_PARAMETERS = "addr-params";
-    private static final String ADDRESS_PARAMETERS_MAPPING = "addr-params-mapping";
-    private static final String DOMAIN_EXTENSION = "domain.extension";
-    private static final String FEATURE_ID = "feature-id";
-    private static final String HOST_EXTENSION = "host.extension";
-    private static final String OPERATION_PARAMETERS = "op-params";
-    private static final String OPERATION_PARAMETERS_MAPPING = "op-params-mapping";
-    private static final String PARAMETERS = "params";
-    private static final String REFERENCES = "refs";
+    public static final String $PROFILE_PREFIX = "$profile.";
```

Comment:
That's not related to the PR per se but I am curious 

@aloubyansky @ehsavoie Starting the field name with `$` is legal in Java but it's the first time I see it. Is there a good reason to use this name instead of plain `PROFILE_PREFIX`?

Reply:
OK, I'm gonna fix that too :).